### DISCUSSION TITLE
i cant get v3 running with Lazy

### DISCUSSION THREAD

**Discussion by xsteadfastx** [lukas-reineke/indent-blankline.nvim#677] [General]
here is my `~/.config/nvim/lua/xsfx/plugins/indent-blankline.lua`

```lua
-- adds indentation guides to all lines (including empty lines)
return {
	{
		"lukas-reineke/indent-blankline.nvim",
		main = "ibl",
		config = function()
			require("ibl").setup({
				indent = {
					char = "#", # was just a test if it does something
				},
			})
		end,
	},
}
```

and it always looks like this:

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/168397/1b3783ec-2473-48b8-816b-8cf0fe034a6b)

in v2 everything looked just the way it should be. im sure its just a little thing i dont understand and that can me be helped with.

thank you

**Comment by lukas-reineke** [comment]
It works like it is supposed to, read `:help ibl.config.indent.tab_char`

**Reply by xsteadfastx** [reply]
haha ok. thank you for your amazing support and plugins!!!
